DESCRIPTION

```diff
@@ -2,7 +2,10 @@ Package: immunedeconv
 Title: Methods for immune cell deconvolution
 Description: Collection of methods for immune cell deconvolution from bulk RNA-seq samples.
 Version: 2.0.4
-Authors@R: person("Gregor", "Sturm", role=c("aut", "cre"), email="mail@gregor-sturm.de")
+Authors@R: c(
+    person("Lorenzo", "Merotto", role=c("aut", "cre"), email="TODO"),
```

Comment:
@LorenzoMerotto, can you please add an up-to-date email address of yours here? I think I only have an outdated one.